fft.py

```diff
@@ -28,15 +28,15 @@
 #=======
 #>>>>>>> 28dff82ecdfe3d39c6bbdf1744410d2af80267a9
 
-def npfft(x,t):
+def npfft(x,delta_t):
     '''Return the normalised amplitude of fft and frequency using fast FFT in Numpy'''
```

Comment:
Just for information. Always make it a habit to write input/output details in comments. Like you did before.